qemu/tests/cfg/uefi_check_resolution.cfg

```diff
@@ -0,0 +1,19 @@
+- uefi_check_resolution:
+    only q35
+    only ovmf
+    only Linux
+    type = uefi_check_resolution
+    timeout = 360
+    boot_menu = on
+    enable_sga = yes
+    boot_menu_hint = "Boot Options"
+    boot_menu_key = "esc;down;kp_enter;down;down;down;down;kp_enter"
+    default_key = "f9;y"
+    change_config = "kp_enter;down"
```

Comment:
We don't really need change_config here, we can construct it in set_change_step().